chapters/scoping.tex

```diff
@@ -135,6 +135,19 @@ \subsection{Composite Name Lookup}\doublelabel{composite-name-lookup}
 \autoref{redeclaration} for further explanations regarding looking inside
 partial packages.
 \end{nonnormative}
+\begin{example}
+Components and classes are part of the same name-space and thus a component may
+not have the same name as its class or the first part of the class-name as that
+would prevent looking up the name of the class.
```

Comment:
I'd rather rephrase this using the term _lookup_; rather than counting the two existing occurrences of _looking up_ as proof that we also use this term for the same thing, I think that those two occurrences should also be rewritten.  Perhaps like this:
```suggestion
would prevent lookup of the class name.
```